.meta/sources/docker.toml

```diff
@@ -60,6 +60,25 @@ A list of image names to match against. If not provided, \
 all images will be included.\
 """
 
+[sources.docker.options.no_auto_partial_merge]
+type = "bool"
+common = false
+default = false
+null = false
+description = """\
+Setting this to `false` will disable the automatic merging of partial events.\
+"""
+
+[sources.docker.options.partial_event_marker]
+type = "string"
+common = false
+default = "_partial"
+null = false
+description = """\
+The field that indicates that the event is partial. A consequent stream of \
+partial events along with the first non-partial event will be merged together.\
+"""
```

Comment:
Well, TOML does not have a `null` value, so there is no way to explicitly set the value to `null`. If this setting is left out it will default to `_partial`, right?

Reply:
Yes. I wasn't aware there's no `null` in TOML. Well, this is a problem. We have to provide a different way to opt-out then.

Reply:
I added a workaround: https://github.com/timberio/vector/pull/1504/commits/f1784a3c8f85ba62652dbfa185ed9a6c9fb87ee1